config/config.go

```diff
@@ -548,6 +548,8 @@ type IsolationRead struct {
 type Experimental struct {
 	// Whether enable global kill.
 	EnableGlobalKill bool `toml:"enable-global-kill" json:"-"`
+	// Whether enable charset feature.
+	EnableCharsetFeat bool `toml:"enable-charset-feat" json:"-"`
```

Comment:
Could we replace the configuration with a hidden global variable？ After this feature GA, we  can delete the variable directly.

Reply:
I think we can not avoid it, this config should not modify online. I suggest using the config field but not expose to users, after the feature GA, we delete it. It is expected that this config is transparent for all.

Reply:
Well, I think that's ok.